src/ucs/vfs/fuse/vfs_fuse.c

```diff
@@ -311,6 +308,18 @@ static ucs_status_t ucs_vfs_fuse_wait_for_path(const char *path)
 #endif
 }
 
+static void ucs_vfs_fuse_thread_reset_affinity()
+{
+    ucs_sys_cpuset_t cpuset;
+    int i;
+    long n;
+
+    for (i = 0, n = sysconf(_SC_NPROCESSORS_ONLN); i < n; ++i) {
+        CPU_SET(i, &cpuset);
+    }
+    ucs_sys_setaffinity(&cpuset);
```

Comment:
check return value from ucs_sys_setaffinity and sysconf, print ucs_diag() in case of failure